services/fluxmonitor.go

```diff
@@ -54,27 +55,58 @@ func ParseFMSpec(jsonSpec json.RawMessage, runtimeConfig store.RuntimeConfig) (F
 	fmConfig.AbsoluteThreshold = gjson.GetBytes(jsonSpec, "absoluteThreshold").Float()
 	fmConfig.RuntimeConfig = runtimeConfig
 
-	var err error
+	if len(fmConfig.Adapters) == 0 {
+		fmSpecErrors = append(fmSpecErrors, "at least one feed url should be specified. Example format: "+
+			"["+
+			"{ \"url\": \"http://localhost:8080\" },"+
+			"{ \"url\": \"http://localhost:8081\" }"+
+			"]")
+	}
+
+	//This is better not be strictly validated so it is flexible for different kind of requests
+	if fmConfig.RequestData == "" {
+		fmSpecErrors = append(fmSpecErrors, "no requestData")
+	}
+	if fmConfig.Multiply == 0 {
+		fmSpecErrors = append(fmSpecErrors, "no precision")
+	}
+	if fmConfig.Threshold <= 0 {
+		fmSpecErrors = append(fmSpecErrors, "'threshold' must be positive")
+	}
+	if fmConfig.AbsoluteThreshold < 0 {
+		fmSpecErrors = append(fmSpecErrors, "'absoluteThreshold' must be non-negative")
+	}
+
+	// this is the case in core
```

Comment:
Thoughts on this @boxhock?

Reply:
Thoughts on what exactly?

Reply:
On the validation that exists on core; one of idleTimer or pollTimer should be enabled.

Reply:
Ah yeah we'll need that, otherwise the feed would never run :sweat_smile: